apps/docker/docker-cmd.ts

```diff
@@ -0,0 +1,174 @@
+import { Command } from "https://cdn.depjs.com/cmd/mod.ts";
+import * as ink from "https://deno.land/x/ink/mod.ts";
+import {
+  Input,
+  Select,
+} from "https://deno.land/x/cliffy@v0.18.2/prompt/mod.ts";
+
+const program = new Command();
+
+program.name("acore.sh docker")
+  .description("Shell scripts for docker")
+  .version("1.0.0");
+
+shellCommandFactory(
+  "start:app",
+  "Startup the authserver and worldserver apps",
+  "docker-compose --profile app up",
+);
+
+shellCommandFactory(
+  "start:app:d",
+  "Startup the authserver and worldserver apps in detached mode",
+  "docker-compose --profile app up -d",
+);
+
+shellCommandFactory(
+  "start:dev",
+  "Startup the dev server",
+  "docker-compose --profile dev up",
+);
+
+shellCommandFactory(
+  "build",
+  "Build the authserver and worldserver",
+  `docker-compose run --rm ac-dev-server bash bin/acore-docker-build`,
```

Comment:
should this be a backtick?

Reply:
Backtick in typescript is used to define strings with the "interpolation" feature enabled. It's exactly like the " " in bash.

Reply:
gotcha. thanks